widget/view.controller.js

```diff
@@ -89,27 +89,18 @@ Copyright end */
         needle.style.transform = `rotate(${needleAngle}deg)`;
       }, 10);
     }
-  
-    function mapRiskColor(riskKey){
-      if (typeof riskKey != 'string') {
-        return '';
-      }
-      switch (riskKey.toLowerCase().trim()) {
-        case 'high':
-          return {'startRiskColor' : '#F66E4F',  'endRiskColor': '#E60C4B'};
-          break;
-        case 'low':
-          return {'startRiskColor' : '#fdfc00',  'endRiskColor': '#fac602'};
-          break;
-        case 'moderate':
-          return {'startRiskColor' : '#07de04',  'endRiskColor': '#6ac359'};
-          break;
-        case 'default':
-          return {'startRiskColor' : '#5596be',  'endRiskColor': '#1d7fbb'};
-        default:
-          return 
 
-      }
+    //map risk color for grid risk column through Confidence picklist value
+    function getRiskScorePicklistColor(riskKey) {
+      const _picklistField = 'IOC Search Confidence';
```

Comment:
Keep picklist field as a param to the function for future use against any picklist.

Picklist can will be provided from edit control